api/reporting/schemas/reporting-codegen.rb

```diff
@@ -26,64 +27,187 @@ def to_output_path(dir_name, table_name, suffix)
   File.expand_path(File.join(dir_name, "#{table_name}.#{suffix}"))
 end
 
-dto_class_name = "BqDto#{to_camel_case(table_name, true)}"
-
-inputs = {
+INPUTS = {
     :describe_csv => to_input_path(File.join(input_dir, 'mysql_describe_csv'), table_name,'csv'),
-    :exclude_columns => to_input_path(File.join(input_dir, 'excluded_columns'), table_name,'txt')
-}
+    :excluded_columns => to_input_path(File.join(input_dir, 'excluded_COLUMNS'), table_name,'txt')
```

Comment:
Was this capitalization intentional?

Reply:
no, I just didn't want to fix it and have to find the dir to delete.

Reply:
I don't understand. The inputs in the PR are in folder `excluded_columns`. And also, aren't you done generating code at this point? I'm not understanding why columns has been capitalized in this file several times